### ISSUE TITLE
Save files with unix linefeeds

### ISSUE THREAD

**Issue by techtonik** [nodebox/nodebox#426]
On Windows nodebox saves files with Windows linefeeds and that creates problem for version control.

**Comment by fdb** [comment]
You mean the .ndbx files?

**Comment by techtonik** [comment]
Yea. I got somewhere around these lines 

https://github.com/nodebox/nodebox/blob/c15874a11f7b90c262afcd2db5f20ea69dc011b2/src/main/java/nodebox/node/NDBXWriter.java#L76

**Comment by techtonik** [comment]
The best way would probably be to set:
```
System.setProperty("line.separator", "\n");
```
I don't know what is the best place for that though.

https://stackoverflow.com/questions/19102804/ensure-unix-style-line-endings-used-when-generating-xml-on-windows-with-java

**Comment by fdb** [comment]
Good call! There is no code there that says "do Windows line feeds", so I'll see how to override the defaults and always use Unix line endings.

Nice find BTW.

**Comment by techtonik** [comment]
@fdb this should do it.
